src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -787,11 +799,11 @@ private void FleeFromPredators(ref WorldPosition position, ref MicrobeAI ai, ref
                 // If the predator is right on top of us there's a chance to try and swing with a pilus
                 ai.MoveWithRandomTurn(2.5f, 3.0f, position.Position, ref control, speciesActivity, random);
             }
-
-            // Sprint until full strain
-            control.Sprinting = true;
         }
 
+        // Sprint until full strain
+        control.Sprinting = true;
```

Comment:
This would probably make prey run from very far away. I could see this helping against some predators, and wearing the prey out or even running into another predator in other cases. Maybe keep the other command while putting this one behind a fear threshold?